### ISSUE TITLE
Vital for accessibility - nice to have for others

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-showcase-examples#41]
It's good to show there are benefits of accessibility for people without disabilities, yet in some places they seem to have equal "weight". I think it's extremely important that we clearly communicate that some things are **vital** for pwds, and nice to have for others.

Examples:
- 1: "  There are lots of reasons why people might need captions – so for content to be available to more people in more places there’s a solution –add captions!" and throughout doesn't communicate that captions are ESSENTIAL for people who are Deaf.
- 2: "  If there’s sufficient color contrast more people can make use of the content in more environments – it’s just common sense" and throughout doesn't adequately communicate that some people cannot use it at all in any environment if there's not sufficient contrast.
- 4: "Thanks to automated readers more people than ever have access to more of the web than ever before – proper coding can improve that even further." and throughout doesn't get across the importance of accessible to screen readers to people who are blind!
- 10

It would be good to go over all scripts and check for this.

Actually, it might be really neat to have a catch phrase that communicates that point (vital for some; beneficial/useful for all/others) that we use in most or all of the videos...

**Comment by nitedog** [comment]
@slhenry - scripts have been revised to address this issue. Do you still have this concern?

**Comment by shawna-slh** [comment]
From quick skim, I'm very happy with the revised scripts addressing this issue!
